src/components/datepicker/calendar-theme.scss

```diff
@@ -28,7 +28,7 @@
   .md-calendar-date-selection-indicator {
     .md-calendar-date.md-focus &,
     &:hover {
-      background: '{{background-300}}';
+      background: '{{background-500-0.32}}';
     }
   }
 
```

Comment:
The comments on Lines 47-50 need to be updated as they are no longer accurate. Perhaps just removing ", even on the dark theme," is enough.

Reply:
```scss
.md-calendar-date-disabled,
.md-calendar-month-label-disabled {
```
These lines seem to also suggest that we should be a calendar demo that shows an enabled and a disabled calendar.

Reply:
You have right https://material.angularjs.org/latest/demo/datepicker
example `Only weekends are selectable`

Reply:
what do You think about
```
.md-calendar-date-disabled,
.md-calendar-month-label-disabled {
  color: '{{foreground-3}}';
}
```
its a litle bit brighter than `background-A200-0.435` in light mode.
But in both modes light/dark looks great.

Reply:
a lot of components use `'{{foreground-3}}'` in `[disabled]` selectors

Reply:
fix was pushed